prow/cluster/tls-ing_ingerss.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: extensions/v1beta1
```

Comment:
in `install-prow.sh` script we deploy another ingress from `starter.yaml`. Is it a problem?